Feature/header

WindyBoss opened the pull request.

WindyBoss committed:
Add header

WindyBoss committed:
Update index.html

MicMinProject reviewed (commented):
W header usun to co masz w head a zostaw tylko body bo head jest juz w pliku projektu

WindyBoss committed:
index changes

WindyBoss committed:
Merge branch 'feature/header' of https://github.com/MicMinProject/goit-project-01 into feature/header

MicMinProject committed:
Update _common.scss

MicMinProject committed:
Update _common.scss

MicMinProject reviewed (commented):
usunąłem tą część, która się bedzie powtarzać

WindyBoss committed:
TItle font

WindyBoss committed:
variables changes

Variables changes

WindyBoss committed:
Update _variables.scss

WindyBoss committed:
Update _variables.scss

WindyBoss committed:
Update _variables.scss

WindyBoss committed:
Main Correction

WindyBoss committed:
Last correction

Added modal window
Adoption to teammate's code
Changed the picture name
Added Favicon

WindyBoss committed:
Create modal.js

MicMinProject committed:
Merge branch 'main' into feature/header
